### PR TITLE
Feature/vta 761

### PR DESCRIPTION
**Pull request by blondeholly**
## Description

Related issue: [VTA-761)

## Before submitting (or marking as "ready for review")

- [ ] Does the pull request title follow the [conventional commit](https://www.conventionalcommits.org/en/v1.0.0/) specification?
- [ ] Have you performed a self-review of the code
- [ ] Have you have added tests that prove the fix or feature is effective and working
- [ ] Did you make sure to update any documentation relating to this change?

### CODE REVIEW

**Review by IuliaDVSA — DISMISSED** [review]

File: src/main/java/util/DVLABasePathFilter.java [hunk 1]
```diff
@@ -18,13 +18,25 @@ public class DVLABasePathFilter implements Filter {
...
         }
         return filterContext.next(filterableRequestSpecification, filterableResponseSpecification);
     }
+
+    /**
+     * Simple method to construct the base URL with the new format following the APIG restructure
+     *
+     * @return Base url as String
+     */
+    private String returnBaseUrl() {
```
Comment (Daniel-Searle) [inline src/main/java/util/DVLABasePathFilter.java:35]: Can you rename this method please; something like createBaseUrl please

File: src/main/java/util/DVLABasePathFilter.java [hunk 2]
```diff
@@ -18,13 +18,25 @@ public class DVLABasePathFilter implements Filter {
...
         if (!isWrongAuth() && !isMissingAuth()) {
             filterableRequestSpecification.header("Authorization", "Bearer " +TokenGenerator.getDVLAToken()).header("X-Api-Key",loader.getApiKeys());
         } else if (isWrongAuth()) {
             filterableRequestSpecification.header("Authorization", "Bearer " + RandomStringUtils.randomAlphanumeric(30));
         }
         return filterContext.next(filterableRequestSpecification, filterableResponseSpecification);
     }
+
+    /**
+     * Simple method to construct the base URL with the new format following the APIG restructure
```
Comment (Daniel-Searle) [inline src/main/java/util/DVLABasePathFilter.java:31]: Please update this description to not include any information about the APIG changes. Just include a description of what the method is doing

File: src/main/resources/loader/develop/test-results_post_payload_trl_761.json [hunk 3]
```diff
@@ -0,0 +1,47 @@
+{
+  "testResultId":"ffd02799-c781-45c8-9fd6-ef5e64b7h6k8",
+  "testStationName":"Rowe, Wunsch and Wisoky",
+  "testStationPNumber":"87-1369569",
+  "testStationType":"gvts",
+  "testerName":"Gica",
+  "testerStaffId":"133",
```
Comment (Daniel-Searle) [inline src/main/resources/loader/develop/test-results_post_payload_trl_761.json:7]: This tester staff id is being used in lots of other tests. Are there any tests that test the amount of test results returned for this given tester staff id? If there is and it is expecting a specific amount of them, they will start failing as a result of this being added

File: src/test/java/testresults/PostTestResultLongSemiTrailer.java [hunk 4]
```diff
@@ -0,0 +1,60 @@
...
+    @Test
+    public void testResultsLongSemiTrailer() {
+        // Read the base test result JSON.
+        String jsonFileName = "test-results_post_payload_trl_761.json";
+        String testResultRecord = GenericData.updateJson(jsonFileName,false);
+        String randomTestResultId = UUID.randomUUID().toString();
+        JsonPathAlteration alterationTestResultId = new JsonPathAlteration("$.testResultId", randomTestResultId, "", "REPLACE");
+
+        // Collate the list of alterations.
+        List<JsonPathAlteration> alterations = new ArrayList<>(Arrays.asList(
```
Comment (Daniel-Searle) [inline src/test/java/testresults/PostTestResultLongSemiTrailer.java:43]: The asList method from Arrays shouldn't be used in this instance as there is only one given parameter. A collections method would be more appropriate for this instance

Comment (Daniel-Searle) [inline src/test/java/testresults/PostTestResultLongSemiTrailer.java:43]: Can this variable instantiation be put onto one line please

**Review by Daniel-Searle — CHANGES_REQUESTED** [review]

**Comment by Daniel-Searle** [comment]
Can the PR description also be updated to add the following: short description of the story, and the link to the story

**Review by Daniel-Searle — APPROVED** [review]

File: src/main/resources/loader/develop/test-results_post_payload_trl_761.json (hunk 3 above)
Comment (blondeholly) [reply src/main/resources/loader/develop/test-results_post_payload_trl_761.json:7]: Have run regression on all packs and no tests are failing on account of this tester staff ID

**Review by IuliaDVSA — APPROVED** [review]

**Pull request merged**